compiler-rt/lib/builtins/riscv/feature_bits.c

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "cpu_model.h"
```

Comment:
Here we have two solution to avoid the build fail

1. Move `feature_bits.c` from `compiler-rt/lib/builtins/riscv/feature_bits.c` to  `compiler-rt/lib/builtins/cpu_model/riscv.c`, and update the `compiler-rt/lib/builtins/CMakeLists.txt`. Then `cpu_model.h` can be searched correctly.  I create a commit for this case https://github.com/BeMg/llvm-project/commit/0aff090de58f1a0920952297fe6f803261186593
2. Simply copy code section from cpu_model.h to here

```
#if __has_attribute(constructor)
#if __GNUC__ >= 9
// Ordinarily init priorities below 101 are disallowed as they are reserved for
// the implementation. However, we are the implementation, so silence the
// diagnostic, since it doesn't apply to us.
#pragma GCC diagnostic ignored "-Wprio-ctor-dtor"
#endif
// We're choosing init priority 90 to force our constructors to run before any
// constructors in the end user application (starting at priority 101). This
// value matches the libgcc choice for the same functions.
#define CONSTRUCTOR_ATTRIBUTE __attribute__((constructor(90)))
#else
// FIXME: For MSVC, we should make a function pointer global in .CRT$X?? so that
// this runs during initialization.
#define CONSTRUCTOR_ATTRIBUTE
#endif
```

Reply:
+1 for option 1

Reply:
I also prefer option 1.  @BeMg Can you take over this review?  I don't have a workable build environment for compiler-rt at the moment.  If you're happy with this change, feel free to take it, apply the change needed for it to build (option 1) and land.  

 I'm in the process of doing a native build of the 19.x release branch (including compiler-rt), but that's likely going to be another ~36 hours or so before it's complete and I can use it for incremental patch testing.